Description: Drop all direct use of flake8
 OpenStack relies on ancient versions of flake8 which
 have a different API to those in Debian and Ubuntu.
 .
 Avoiding running any unit tests that make use
 of flake8.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

--- a/neutron/tests/unit/hacking/test_checks.py
+++ b/neutron/tests/unit/hacking/test_checks.py
@@ -12,7 +12,6 @@
 
 import re
 
-from flake8 import engine
 from hacking.tests import test_doctest as hacking_doctest
 import pkg_resources
 import pycodestyle
@@ -251,6 +250,7 @@
 
     def test_pycodestyle(self):
 
+        self.skipTest('Disabled until hacking supports pycodestyle/flake8')
         # NOTE(jecarey): Add tests marked as off_by_default to enable testing
         turn_on = set(['H106'])
         if self.options.select:
@@ -287,7 +287,7 @@
         yield (line, match.groups())
 
 
-def load_tests(loader, tests, pattern):
+def _load_tests(loader, tests, pattern):
 
     default_checks = [e.name for e
                       in pkg_resources.iter_entry_points('flake8.extension')]
